gapic/src/main/com/google/gapid/views/TracerDialog.java

```diff
@@ -257,6 +259,19 @@ protected void buttonPressed(int buttonId) {
       private static final int DEFAULT_START_FRAME = 100;
       private static final String PERFETTO_LABEL = "Profile Config: ";
 
+      private static enum StartType {
+        Beginning, Manual, Frame;
+      }
+
+      private static final HashMap<Integer, String> startTypeTitles = new HashMap<Integer, String>();
+
+      static
+      {
+        startTypeTitles.put(StartType.Beginning.ordinal(), "Beginning");
+        startTypeTitles.put(StartType.Manual.ordinal(), "Manual");
+        startTypeTitles.put(StartType.Frame.ordinal(), "Frame");
+      };
+
```

Comment:
Just use `startType.name()`. If, however, you don't want to tie yourself to the name of the enum, add a final field in the enum for it. Something like:
```
private static enum StartType {
  Beginning("Beginning"), ... ;

  public final String display;

  private StartType(...) { ... }
}
```